include/live_model.h

```diff
@@ -153,7 +153,7 @@ namespace reinforcement_learning {
     int request_decision(const char * context_json, unsigned int flags, decision_response& resp, api_status* status = nullptr);
```

Comment:
Can you also add the deprecated attribute similar to what we do in VW? https://github.com/VowpalWabbit/vowpal_wabbit/blob/master/vowpalwabbit/vw.h#L42